.github/workflows/validation.yml

```diff
@@ -0,0 +1,37 @@
+name: Validation
+
+on: push 
+
+jobs:
+  lint:
+    name: Linting
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: Install dependencies
+        run: |
+          npm install
+      - name: ESLint
+        run: npm run lint
+
+  test:
+    name: Run unit tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: Install dependencies
```

Comment:
https://github.com/bahmutov/npm-install this provides some better cacheing

Reply:
There is official cache action:
https://github.com/graphql/graphql-js/blob/cd273ad136d615b3f2f4c830bd8891c7c5590c30/.github/workflows/ci.yml#L18-L24
I think we should minimize the number of 3rd-party dependencies especially since this particular one is not very popular.

Reply:
done